webapp/src/reducer.js

```diff
@@ -60,11 +60,21 @@ function rhsPluginAction(state = null, action) {
     }
 }
 
+function rhsState(state = null, action) {
+    switch (action.type) {
+    case UPDATE_RHS_STATE:
+        return action.state;
+    default:
+        return state;
+    }
+}
+
 export default combineReducers({
     rootModalVisible,
     postID,
     issues,
     inIssues,
     outIssues,
+    rhsState,
```

Comment:
I still do not see how this state changes the behaviour on the RHS. Am I missing anything?

Also, take in mind that the RHS only have two different tabs (my Todos and sent Todos). Incoming Todos are in the "my Todos" tab. A nice solution could be to use this state to collapse the incoming Todos or the rest of Todos depending on it.

Reply:
I intended to open the `Sent` part of RHS when I click on `Todos Outgoing` sidebar button. Do you require it or shall I remove it. It will require some more changes to state and will modify the RHS component to get data from redux.

Reply:
Nevermind, I have done it.
On clicking `Your Todos` or `Incoming Todos`, you see the `Todos` tab
On clicking `Outgoing Todos`, you see the `Sent` tab

Reply:
@larkox can you confirm if the changes I made are acceptable?